Content.Client/Administration/UI/Notes/AdminNotesLinePopup.xaml.cs

```diff
@@ -49,17 +57,25 @@ private void DeletePressed(ButtonEventArgs args)
         if (!ConfirmingDelete)
         {
             ConfirmingDelete = true;
+            Task.Delay(3000).ContinueWith(_ => ResetDeleteButton()); // TODO: fix
```

Comment:
? This looks very hacky.

Reply:
It ended up not working anyway, hence the TODO. Forgot to fix it in the end, I'm not even sure what else to try